### DISCUSSION TITLE
Checksum error on uptime kuma.

### DISCUSSION THREAD

**Discussion by mjacobs63** [community-scripts/ProxmoxVE#195] [General]
I get a checksum error every time I run the install script for uptime kuma.  Immediately following the download the error is displayed.  Is there a problem with the source file?

**Comment by mjacobs63** [comment]
![image](https://github.com/user-attachments/assets/f81c0b2c-c259-4804-9a70-be4813b5a1cf)

**Comment by MickLesk** [comment]
Post full Install with verbose please

**Reply by mjacobs63** [reply]
__  __      __  _                   __ __                     
  / / / /___  / /_(_)___ ___  ___     / //_/_  ______ ___  ____ _
 / / / / __ \/ __/ / __  __ \/ _ \   / ,< / / / / __  __ \/ __  /
/ /_/ / /_/ / /_/ / / / / / /  __/  / /| / /_/ / / / / / / /_/ / 
\____/ .___/\__/_/_/ /_/ /_/\___/  /_/ |_\__,_/_/ /_/ /_/\__,_/  
    /_/                                                          
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: ********
Container ID: 127
Using Hostname: uptimekuma
Using Disk Size: 4
Allocated Cores: 1
Allocated RAM: 1024
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: no
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using Vlan: Default
Enable Root SSH Access: yes
Enable Verbose Mode: yes
Creating a Uptime Kuma LXC using the above advanced settings
 ✓ Using Vault for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 -checksum mismatch: got '1f9978f1e266b7d7e4bf0b7245eff565a75dda3d6a9ce8095379ef40ed3ef11c02ecb5ff851c55e6749dcbd283bc4c52782dae9ea8cce35e8684c356164724c6' != expect '39f6d06e082d6a418438483da4f76092ebd0370a91bad30b82ab6d0f442234d63fe27a15569895e34d6d1e5ca50319f62637f7fb96b98dbde4f6103cf05bff6d'
 ✓ Downloaded LXC Template
 /unable to create CT 127 - volume 'Vault:vztmpl/debian-12-standard_12.7-1_amd64.tar.zst' does not exist
 -bash: line 182: exit: A problem occured while trying to create container.: numeric argument required

**Reply by MickLesk** [reply]
Thats Not an uptimekuma Error. 

You do anything with Advanced Settings who the script failed. I think you enter a wrong LXC ID or something

At my test setup it runs fine with basic settings and Advanced.

**Reply by mjacobs63** [reply]
The LXC ID was just the next sequential number.  I used the same advanced settings on this as I do on the majority of my other LXCs.  Taking another look at it, it appears that there is a different version of the Debian 12 template in the "Vault" than what it is looking for.  Maybe if I just delete the existing one it will download a new one.

**Reply by mjacobs63** [reply]
That was it.  It was looking for debian-12-standard_12.7-1_amd64.tar.zst but was finding debian-12-standard_12.2-1_amd64.tar.zst.  When I renamed the existing file and retried, it downloaded the correct template and the install completed successfully.  Thanks for your help.
